homeassistant/components/weheat/sensor.py

```diff
@@ -142,6 +148,53 @@ class WeHeatSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.TOTAL_INCREASING,
         value_fn=lambda status: status.energy_total,
     ),
+    WeHeatSensorEntityDescription(
+        translation_key="energy_output",
+        key="energy_output",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        value_fn=lambda status: status.energy_output,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="compressor_rpm",
+        key="compressor_rpm",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
+        value_fn=lambda status: status.compressor_rpm,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="compressor_percentage",
+        key="compressor_percentage",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda status: status.compressor_percentage,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="indoor_unit_water_pump_state",
+        key="indoor_unit_water_pump_state",
+        value_fn=lambda status: status.indoor_unit_water_pump_state,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="indoor_unit_auxiliary_pump_state",
+        key="indoor_unit_auxiliary_pump_state",
+        value_fn=lambda status: status.indoor_unit_auxiliary_pump_state,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="indoor_unit_dhw_valve_or_pump_state",
+        key="indoor_unit_dhw_valve_or_pump_state",
+        value_fn=lambda status: status.indoor_unit_dhw_valve_or_pump_state,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="indoor_unit_gas_boiler_state",
+        key="indoor_unit_gas_boiler_state",
+        value_fn=lambda status: status.indoor_unit_gas_boiler_state,
+    ),
+    WeHeatSensorEntityDescription(
+        translation_key="indoor_unit_electric_heater_state",
+        key="indoor_unit_electric_heater_state",
+        value_fn=lambda status: status.indoor_unit_electric_heater_state,
+    ),
```

Comment:
What are the possible states of these entities?

Reply:
`true`, `false` or `null`

Reply:
In that case we should create binary sensors out of those, as well, the state is binary. So let's remove them from this PR and add them as binary sensors

Reply:
Do you want to split the PR or update them in this one?

Reply:
Split the PR please :)

Reply:
I've removed the state sensors from this PR